config/datasets/gcp.yaml

```diff
@@ -21,6 +21,11 @@ catalog_ancestries: ${datasets.inputs}/v2d/gwas-catalog-v1.0.3-ancestries-r2022-
 catalog_sumstats_lut: ${datasets.inputs}/v2d/harmonised_list.txt
 finngen_phenotype_table_url: https://r9.finngen.fi/api/phenos
 ukbiobank_manifest: gs://genetics-portal-input/ukb_phenotypes/neale2_saige_study_manifest.190430.tsv
+ukbiobank_sumstat_path: gs://genetics-portal-dev-analysis/hn9/sumstats/100001_raw.neale2.gwas.imputed_v3.both_sexes.GRCh38.tsv.gz
+ukbiobank_sumstats_dirs:
+  - gs://genetics-portal-dev-analysis/hn9/neale_sumstats
+  - gs://genetics-portal-dev-analysis/hn9/saige_sumstats
+
```

Comment:
Great! In this case I suggest we keep this discussion thread open until the values are replaced with the final ones, to avoid accidentally merging the temporary ones